INSTALL.txt

```diff
@@ -0,0 +1,27 @@
+Installation instructions
+=========================
+
+Minimal tools needed
+--------------------
+
+The minimal RPMs (as provided by CentOS7 yum) needed to build this package are:
+autoconf automake make libtool libtool-ltdl libtool-ltdl-devel gcc gcc-c++ patch openssl openssl-devel
```

Comment:
At the moment, you also need curl because it's used for downloading the gsi-openssh patches. IIRC, this happens in the autoreconf command, i.e. before you get the chance to specify --disable-gsi-openssh.

Alternatively, you can tell people to delete the "prep-gsissh" script if they don't want to build gsi-openssh.

Reply:
Pretty sure I did not need to install curl explicitly.
Maybe it came with the base CentOS7 install.
If you know it is needed, feel free to add it.